src/components/Select/Select.stories.tsx

```diff
@@ -78,7 +78,7 @@ export const Suggestion = () => (
   />
 )
 
-export const Aynsc = () => (
+export const Asynsc = () => (
```

Comment:
sei que tu arrumou ali também, mas não seria `Async` ?

Reply:
```suggestion
export const Async = () => (
```
Arrumei uma letra e esqueci da outra kkkk